libs/components/tabs/src/lib/modules/tabs/tab.component.ts

```diff
@@ -61,14 +61,15 @@ export class SkyTabComponent implements OnChanges, OnDestroy {
    * This input only applies when the tabset's `tabStyle` is `"tabs"`.
    */
   @Input()
-  public set permalinkValue(value: string) {
-    this._permalinkValue = this.permalinkService.urlify(value);
+  public set permalinkValue(value: string | undefined) {
+    this.#permalinkValue = value;
+    this.#permalinkValueOrDefault =
```

Comment:
I went with a `OrDefault` variable here as the default relies on another variable that could change at a different time.